content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -128,6 +128,11 @@ spec:
   volumeSnapshotClassName: csi-hostpath-snapclass
   source:
     persistentVolumeClaimName: pvc-test
+status:
+  boundVolumeSnapshotContentName: snapcontent-9ef06180-0665-4b1d-ae02-a1d35f74b262
+  creationTime: "2023-02-16T23:03:18Z"
+  readyToUse: true
+  restoreSize: 1Gi
```

Comment:
I think "status" should not be included in a yaml file.  If you are showing an example of an already created VolumeSnapshot, then it is appropriate to show the status as well.  This is an example yaml file.